featurebyte/schema/worker/task/feature_list_create.py

```diff
@@ -0,0 +1,40 @@
+"""
+Feature list creation creation schema
+"""
+from __future__ import annotations
+
+from typing import List
+
+from pydantic import BaseModel, Field, StrictStr
+
+from featurebyte.enum import ConflictResolution, WorkerCommand
+from featurebyte.models.base import PydanticObjectId
+from featurebyte.models.feature_list import FeatureListModel
+from featurebyte.schema.worker.task.base import BaseTaskPayload, TaskType
+
+
+class FeatureParameters(BaseModel):
+    """Feature parameters"""
+
+    id: PydanticObjectId
+    name: StrictStr
```

Comment:
In the new feature list creation task, each feature should be already saved, is that right? Is the name field redundant given the id?

Reply:
Oh I guess it's to handle conflict resolution in the SDK.

Reply:
Yes. As per our offline discussion, will add an option to create feature list using feature ids only.